src/test/resources/mockito-extensions/org.mockito.plugins.MockMaker

```diff
@@ -0,0 +1 @@
+mock-maker-inline
```

Comment:
This has broken a bunch of other tests so I might have to remove it and re-write the contributor tests the old way